nilearn/decoding/fista.py

```diff
@@ -101,6 +104,8 @@ def mfista(f1_grad, f2_prox, total_energy, lipschitz_constant, w_size,
         float, and a dict with the key "converged", that says if the method to
         compute f2_prox converged or not.
 
+    dgap_factor: ???? TODO: Add description.
```

Comment:
This is a technical parameter, the dual gap factor. used for debugging purpose (control of the convergence)